app/src/main/java/com/asfoundation/wallet/widget/SystemView.java

```diff
@@ -142,6 +142,13 @@ public void showEmpty(View view) {
     emptyBox.addView(view);
```

Comment:
Unrelated again, but lots of empty files to remove (TransactionsAdapter, AppCoinsApplicationViewHolder, TransactionsViewModel).